Specs/karma-main.js

```diff
@@ -20,12 +20,14 @@
 
     if (release) {
         require.config({
-            baseUrl : '/base/Build/Cesium'
+            baseUrl : '/base/Build/Cesium',
+            waitSeconds : 0
```

Comment:
Should we keep these changes to karma-main.js?

Reply:
Yes, discussed with @mramato so the browser doesn't hang when loading modules.